modules/compactor/compactor.go

```diff
@@ -149,7 +149,8 @@ func (c *Compactor) Owns(hash string) bool {
 }
 
 func (c *Compactor) Combine(objA []byte, objB []byte) []byte {
-	return tempo_util.CombineTraces(objA, objB)
+	combinedTrace, _ := tempo_util.CombineTraces(objA, objB)
```

Comment:
let's log this error

Reply:
Hmm, as of now it will be logged at `CombineTrace` and in the parent fn, should we remove logging at `CombineTrace` altogether?

Reply:
yeah, if we're going to return an error from CombineTraces(), let's log it outside that function.  honestly it's weird that a utility function like CombineTraces logs anything.